bundles/org.dataflowanalysis.analysis.converter/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.json
```

Comment:
Do we still need this `.gitignore`? A better approach would be to move all artifact files to the testmodel project, test as much as possible without actually saving files and then (only if required) adding a separate gitignore there

Reply:
There is another comment describing how to approach artifact files in tests below, see there for the details